lucille-core/pom.xml

```diff
@@ -305,6 +305,42 @@
           </archive>
         </configuration>
       </plugin>
+      <plugin>
```

Comment:
looks like this still needs the config added to make sure we're using the jar

       <configuration>
          <classesDirectory>target/fake</classesDirectory>

etc.